packages/SystemsThinkingWorkbench-Core.package/STWTransferConnection.class/instance/performSimulationStep.st

```diff
@@ -3,10 +3,11 @@ performSimulationStep
 	
 	| particle |
 	
-	self startNode level: self startNode level - self intake.
-	particle := STWParticle new center: (self position + self segment start); color: self startNode fillingMorph color.
-	self particles add: particle.
-	self addMorph: particle.
-	self lastStepTime ifNil: [
-		self startStepping.
-		self lastStepTime: DateAndTime millisecondClockValue.].
+	(self endNode capacity ifNil: [true] ifNotNil: [(self endNode level) < (self endNode capacity)]) ifTrue: [
```

Comment:
See negative level pull request.

**You shall hereby be permitted to merged negative and maximum capacity PRs to apply changes to both.**